R-package/src/install.libs.R

```diff
@@ -196,6 +246,7 @@ if (!use_precompile) {
   }
 
   # build the library
+  message("building lib_lightgbm")
```

Comment:
`message("building lib_lightgbm")`

For consistency, use upper case:

`message("Building lib_lightgbm")`

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3065/commits/d1d8022cc36b5bff616192474a938861fcb7afce